src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -128,7 +128,7 @@ export class PoolFormComponent extends CdForm implements OnInit {
       maxBlobSize: new FormControl('', {
         updateOn: 'blur'
       }),
-      ratio: new FormControl('', {
+      ratio: new FormControl('0.875', {
```

Comment:
If this is a constant default value, lets initialize it at the start or outside the class as
`const DEFAULT_RATIO = 0.875` and use the constant here!

Reply:
Done!